common/build/eslint-plugin-fluid/src/test/mockFiles/no-unchecked-record-access/fileWithOnlyRecordAccess.ts

```diff
@@ -16,57 +16,61 @@ type StaticType = { a: string; b: string };
 type IndexSignatureType = { [key: string]: string };
 
 const nestedObj: NestedIndexProps = { nested: { a: "hello" } };
-nestedObj.nested.a; // ok: Accessing nested property 'a', 'a' is defined
+nestedObj.nested.a; // ok: Accessing nested index property 'a' without requiring a particular result
 nestedObj.nested.a.length; // defect: Accessing length of a nested undefined property
```

Comment:
Comment says property is undefined, but it is only possibly missing (possibly undefined result).